src/components/learner-credit-management/data/hooks/useBudgetDetailTabs.jsx

```diff
@@ -19,9 +21,16 @@ export const useBudgetDetailTabs = ({
   MembersTabElement,
 }) => {
   const intl = useIntl();
-  const isRetired = subsidyAccessPolicy?.retired;
+  const { status } = getBudgetStatus({
+    intl,
+    startDateStr: subsidyAccessPolicy?.subsidyActiveDatetime,
+    endDateStr: subsidyAccessPolicy?.subsidyExpirationDatetime,
+    isBudgetRetired: subsidyAccessPolicy?.retired,
+  });
+  const disbleCatalogTab = status === BUDGET_STATUSES.retired || status === BUDGET_STATUSES.expired;
```

Comment:
nit: typo in `disable`

nit x2: perhaps `isCatalogTabDisabled` to be more explicit/obvious this is a boolean.

nit x3: similar to other comments, perhaps `[BUDGET_STATUSES.retired, BUDGET_STATUSES.expired].includes(status)` to simplify the conditional slightly.